apps/web/src/components/MdxComponents.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";

```

Comment:
Let's stick with the default variables for `h1`, `h2`,... and just keep the `a` tag as it is a special case.

Reply:
Okay, I'll keep the default styling.